lib/autofix.js

```diff
@@ -186,26 +192,26 @@ function formatFileContent(options, file, filePath) {
   if (result.error) {
     if (result.error.code === 'ENOENT') {
       if (hasElmFormatPathFlag) {
-        throw new ErrorMessage.CustomError(
-          'ELM-FORMAT NOT FOUND',
-          // prettier-ignore
-          `I could not find the executable for ${chalk.magentaBright('elm-format')} at the location you specified:
+        throw hasElmFormatPathFlag
+          ? new ErrorMessage.CustomError(
```

Comment:
I have to admit this was a bit jarring :sweat_smile: I hadn't seen `throw conditon ? a : b` before, and it wasn't clear that it didn't just do something like `throw hasElmFormatPathFlag ; <something else>`

Maybe simpler to have something like:

```js
const details = hasElmFormatPathFlag
  ? `I could not find the executable for ${chalk.magentaBright('elm-format')} at the location you specified: ...`
   : `the other message`;

throw new ErrorMessage.CustomError(..., details);
```

What do you think?

Reply:
Oh, I'd meant to change this. Thanks for catching that!